reactivesocket-cpp/src/Frame.cpp

```diff
@@ -30,6 +29,10 @@ std::unique_ptr<folly::IOBuf> FrameBufferAllocator::allocateBuffer(
   return folly::IOBuf::createCombined(size);
 }
 
+std::unique_ptr<folly::IOBufQueue> FrameBufferAllocator::allocateQueue() {
```

Comment:
It's there for consistency and to keep allocations together. Otherwise the logic of providing queue options would have to get repeated unnecessarily. Since it's not detrimental performance-wise, I see no harm adding this utility function.